docker-compose.yml

```diff
@@ -28,8 +29,26 @@ services:
     ports: ['${HOST_BIND_DB_PORT}:5432']
     restart: on-failure
 
+  minio:
+    # https://docs.min.io/docs/minio-docker-quickstart-guide.html
+    image: minio/minio
+    command: server /minio-data
+    environment:
+      - MINIO_ACCESS_KEY=minio_access_key
+      - MINIO_SECRET_KEY=minio_secret_key
+      - MINIO_HTTP_TRACE=/minio-data/requests.log
+    # we don't have to expose it directly to the host machine for our demo,
+    # but it's useful for starting service on the host
+    ports: ['9000:9000']
```

Comment:
are you sure we need this port global - on the host machine? we have a number of minios in the final demo setup, and I tend to hide them because they will conflict otherwise...

Reply:
I'm not, but it's more convenient to run tests from PyCharm without setting up docker interpreter. So far I've added env var to define host port. Let's just disable the mapping when it's become a pain.